budcamp/Homework2/Homework2/Point.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    struct Point
+    {
+        public double X { get; set; }
+        public double Y { get; set; }
+
+        public Point()
+        {
+            X = 0;
+            Y = 0;
+        }
+
+        public Point(int x, int y)
+        {
+            this.X = x;
+            this.Y = y;
+        }
+
+        public bool TryParsePoint(string input, out Point point)
+        {
+            point = default(Point);
+
+            try
```

Comment:
if you're using TryParse, then there is no much sense in try..block - exception won't be thrown anyway, right?

Reply:
done